.github/workflows/coding-standards.yml

```diff
@@ -1,101 +1,101 @@
+            - name: Run PHPCS on all Core files
+              run: vendor/bin/phpcs --standard=phpcs.ruleset.xml --extensions=php --colors -s -p -v .
 
-      - name: Check test suite files for warnings
-        run: vendor/bin/phpcs tests --standard=phpcs.ruleset.xml --extensions=php --colors -s -p -v .
+            - name: Check test suite files for warnings
+              run: vendor/bin/phpcs tests --standard=phpcs.ruleset.xml --extensions=php --colors -s -p -v .
 
-  # Runs the JavaScript coding standards checks.
-  #
-  # Performs the following steps:
-  # - Checks out the repository.
-  # - Logs debug information about the runner container.
-  # - Installs NodeJS 12 (todo: install the version of NPM specified in the `.nvmrc` file to support older branches)
-  # - Sets up caching for NPM.
-  # - Logs updated debug information.
-  # _ Installs NPM dependencies.
-  # - Run the WordPress JSHint checks.
-  # - todo: Configure Slack notifications for failing tests.
-  jshint:
-    name: JavaScript Coding Standards
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v2
+    # Runs the JavaScript coding standards checks.
+    #
+    # Performs the following steps:
+    # - Checks out the repository.
+    # - Logs debug information about the runner container.
+    # - Installs NodeJS 12 (todo: install the version of NPM specified in the `.nvmrc` file to support older branches)
+    # - Sets up caching for NPM.
+    # - Logs updated debug information.
+    # _ Installs NPM dependencies.
+    # - Run the WordPress JSHint checks.
+    # - todo: Configure Slack notifications for failing tests.
+    jshint:
+        name: JavaScript Coding Standards
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout repository
+              uses: actions/checkout@v2
 
-      - name: Log debug information
-        run: |
-          npm --version
-          node --version
-          git --version
-          php --version
-          composer --version
+            - name: Log debug information
+              run: |
+                  npm --version
+                  node --version
+                  git --version
+                  php --version
+                  composer --version
 
-      - name: Install NodeJS
-        uses: actions/setup-node@v2
-        with:
-            node-version-file: '.nvmrc'
+            - name: Install NodeJS
+              uses: actions/setup-node@v2
+              with:
+                  node-version-file: '.nvmrc'
 
-      - name: NPM install
-        run: npm ci --legacy-peer-deps
+            - name: NPM install
+              run: npm ci --legacy-peer-deps
 
-      - name: Run JSHint
-        run: npm run lint:js
-        if: ${{ success() || failure() }}
+            - name: Run JSHint
+              run: npm run lint:js
+              if: ${{ success() || failure() }}
```

Comment:
please revert these. not sure why they are changing.

Reply:
Additional notes on this:

<img width="682" alt="Screenshot 2023-09-10 at 5 37 40 PM" src="https://github.com/GatherPress/gatherpress/assets/818980/e186e79c-b5a4-4aeb-97e4-d49b105897d3">